extensions/fisher/README.md

```diff
@@ -0,0 +1,66 @@
+# Fisher Plugin Manager for Raycast
+
+A Raycast extension to help you manage your Fish shell plugins with [Fisher](https://github.com/jorgebucaran/fisher).
+
+## ✨ Features
+
+- 🔍 **Search & Install Plugins** – Browse from a curated registry and install with one click
+- 📦 **List Installed Plugins** – View all installed plugins with descriptions
+- 🔧 **Update Plugins** – Update individual plugins or all at once
+- ❌ **Remove Plugins** – Easily remove plugins with confirmation
+- 📄 **Plugin Details** – View descriptions, GitHub links, stars, and last updated date
+
+## 📦 Plugin Metadata
+
+Plugin info is powered by a local JSON registry that you can update manually. It includes name, description, and GitHub repo.
+
+## 🛠️ Setup (Optional for Development)
+
+To increase GitHub API rate limits during development:
+
+1. Create a `.env` file in the project root:
+
+   ```javascript
+   GITHUB_TOKEN = your_personal_token_here;
+   ```
+
+2. Install dependencies:
+
+   ```bash
+   npm install dotenv
+   ```
+
+3. On `PluginDetail` component:
+
+   ```javascript
+   const headers = {
+     Authorization: `Bearer ${process.env.GITHUB_TOKEN}`,
+     Accept: "application/vnd.github+json",
+   };
+
+   const [metaRes, relRes] = await Promise.all([
+     fetch(`https://api.github.com/repos/${owner}/${repo}`, { headers }),
+     fetch(`https://api.github.com/repos/${owner}/${repo}/releases/latest`, { headers }),
+   ]);
+   ```
+
+The extension uses this token to authenticate GitHub API requests when available.
```

Comment:
This is not a proper solution for users who install this extension from the store.

You can consider exposing an [extension preference](https://developers.raycast.com/api-reference/preferences) to allow users to input their GitHub token.